css/mini-course-dialog.styl

```diff
@@ -1,3 +1,71 @@
+.modal-form-underlay.mini-course-dialog
+  background-color: rgba(0,0,0,0.6)
+
 .mini-course-dialog
-  .step-through-controls
-    display: none
\ No newline at end of file
+  padding: 0
+  
+  &:not(.modal-form-underlay)
+    border: solid 0.5em MINI_COURSE_BLUE
+    border-radius: 8px
+
+  @media screen and (max-width: 400px)
+    margin-left: 0em
+    margin-top: 1.75em
+        
+  &__steps
+    display: flex
+    flex-direction: column
+    max-height: 75vh
+    width: 75vw
+      
+    .steps__step
+      overflow-y: auto
+      
+      &-actions
+        align-items: center
+        display: flex
+        justify-content: space-between
+        
+        .action__opt-out
+          color: #a1a1a1
+          font-style: italic
+          font-size: 0.75em
+          margin: 0 0.25em
+          
+        .action__buton
+          background-color: MINI_COURSE_BLUE
+          
+  .media-card.steps__step
+    > .media-card-content
+      margin: 1em 3vw
+      
+      .markdown
+        h1, h2, h3, h4, h5
+          color: MINI_COURSE_BLUE
+          font-size: 1.17em
+          
+        // Being co-opted for the image credit. TODO: Support figcaption for the media in tutorials and mini-courses  
+        h6
+          color: #a1a1a1
+          font-style: italic
+          margin: 0 0 0.5em
+          text-align: right
+          width: 100%
+          
+          @media screen and (max-width: 400px)
+            text-align: center
+          
+        p
+          color: #2b2b2b
+          font-size: 0.9em
+
+        a
+          color: MINI_COURSE_BLUE
+        
+  .media-card-header
+    background-color: #111111
+    
+  .media-card-media
+    display: block
+    margin: 0 auto
+    max-height: 33vh
```

Comment:
How about a `max-width: 100%` so a wide image doesn't cause the dialog to scroll?